static/js/tutorials.js

```diff
@@ -0,0 +1,118 @@
+const { TypesenseInstantSearchAdapter, instantsearch } = window;
+const observer = lozad();
+
+const typesenseInstantsearchAdapter = new TypesenseInstantSearchAdapter({
+  server: {
+    apiKey: "oRW875O3vjeV3qX4ENl1iIA0u2IRDbTQ", // Be sure to use an API key that only allows search operations
+    nodes: [
+      {
+        host: "cgnvrk0xwyj9576lp-1.a1.typesense.net",
+        port: "443",
+        protocol: "https",
+      },
+    ],
+    cacheSearchResultsForSeconds: 2 * 60, // Cache search results from server. Defaults to 2 minutes. Set to 0 to disable caching.
+  },
+  // The following parameters are directly passed to Typesense's search API endpoint.
+  //  So you can pass any parameters supported by the search endpoint below.
```

Comment:
This seems useful but I would suggest having more info here if this tip will be important for understanding search functionality in the future - is this endpoint what people are using/querying in the search bar of docs? and code here is for that?

Reply:
the search bar works similar but with a different api key and different parameters. It's two different collections that the data for searching is stored in. This code dynamically adds the search sidebar and the tutorial cards to the page by accessing the collection and searching for tutorials that fit the search criteria and then adding them to the page